app/src/main/java/com/github/factotum_sdp/factotum/ui/display/utils/PhotoDiffCallback.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.factotum_sdp.factotum.ui.display.utils
+
+import androidx.recyclerview.widget.DiffUtil
+import com.google.firebase.storage.StorageReference
+
+// Callback to calculate the difference between two photo items
+class PhotoDiffCallback : DiffUtil.ItemCallback<StorageReference>() {
+    override fun areItemsTheSame(oldItem: StorageReference, newItem: StorageReference): Boolean {
+        return oldItem.path == newItem.path
```

Comment:
Probably can easily implement a test, that checks whether two photo items are the same or not.